build.cake

```diff
@@ -42,7 +43,63 @@ var password = EnvironmentVariable("CAKE_GITHUB_PASSWORD");
 
 Setup(() =>
 {
-    Information("Building version {0} of Cake.", semVersion);
+    if (isRunningOnWindows)
+    {
+        Information("Calculating Semantic Version");
+        if (AppVeyor.IsRunningOnAppVeyor)
```

Comment:
This might not be enough.  Given that we are running on multiple build servers, i.e. Jenkins, TeamCity, Bamboo, etc, we want GitVersion to be ran with `/output buildserver` so that it can do it's branch normalisation, where required.  Perhaps what we need here is a check to see if we are **not** running locally.